### ISSUE TITLE
Cannot destructure property 'nuxt' of 'this' as it is undefined

### ISSUE THREAD

**Issue by chasebank** [nuxt/image#1213]
Got this with a fresh install of Nuxt 3 locally, reproduced in Stackblitz: https://stackblitz.com/edit/nuxt-starter-2ssmm6

Following [v1 installation guide](https://image.nuxt.com/get-started/installation)

yields:
```
 ERROR  Cannot start nuxt:  Cannot destructure property 'nuxt' of 'this' as it is undefined.                    7:21:36 PM

  at imageModule2 (node_modules/@nuxt/image/dist/module.cjs:222:11)
  at Module.installModule (node_modules/@nuxt/kit/dist/index.mjs:2499:101)
  at async initNuxt (node_modules/nuxt/dist/index.mjs:3875:7)
  at async NuxtDevServer._load (node_modules/nuxi/dist/chunks/dev2.mjs:268:5)
  at async NuxtDevServer.load (node_modules/nuxi/dist/chunks/dev2.mjs:200:7)
  at async NuxtDevServer.init (node_modules/nuxi/dist/chunks/dev2.mjs:195:5)
  at async Object.run (node_modules/nuxi/dist/chunks/dev-child.mjs:116:5)
  at async runCommand$1 (node_modules/nuxi/dist/shared/nuxi.4fde776c.mjs:1678:16)
  at async runCommand$1 (node_modules/nuxi/dist/shared/nuxi.4fde776c.mjs:1669:11)
  at async runMain$1 (node_modules/nuxi/dist/shared/nuxi.4fde776c.mjs:1803:7)
```

**EDIT**:
Just noticed #1212 and sure enough, 0.7.2 is being installed. Versioning issue?? Rolling back to 1.3.0 produces dozens of install warns, but appears to resolve this "cannot destructure property 'nuxt'" error.

```
warning vscode-languageclient@7.0.0: The engine "vscode" appears to be invalid.
warning sharp@0.33.2: The engine "libvips" appears to be invalid.
warning @img/sharp-darwin-arm64@0.33.2: The engine "pnpm" appears to be invalid.
warning @img/sharp-darwin-arm64@0.33.2: The engine "glibc" appears to be invalid.
warning @img/sharp-libvips-linux-arm@1.0.1: The engine "pnpm" appears to be invalid.
warning @img/sharp-libvips-linux-arm@1.0.1: The engine "glibc" appears to be invalid.
warning @img/sharp-libvips-linux-arm64@1.0.1: The engine "pnpm" appears to be invalid.
warning @img/sharp-libvips-linux-arm64@1.0.1: The engine "glibc" appears to be invalid.
warning @img/sharp-libvips-darwin-x64@1.0.1: The engine "pnpm" appears to be invalid.
warning @img/sharp-libvips-darwin-x64@1.0.1: The engine "macos" appears to be invalid.
warning @img/sharp-libvips-linuxmusl-arm64@1.0.1: The engine "pnpm" appears to be invalid.
warning @img/sharp-libvips-linuxmusl-arm64@1.0.1: The engine "musl" appears to be invalid.
warning @img/sharp-libvips-linux-s390x@1.0.1: The engine "pnpm" appears to be invalid.
warning @img/sharp-libvips-linux-s390x@1.0.1: The engine "glibc" appears to be invalid.
warning @img/sharp-linux-arm64@0.33.2: The engine "pnpm" appears to be invalid.
warning @img/sharp-linux-arm64@0.33.2: The engine "glibc" appears to be invalid.
warning @img/sharp-linux-arm@0.33.2: The engine "pnpm" appears to be invalid.
warning @img/sharp-linux-arm@0.33.2: The engine "glibc" appears to be invalid.
warning @img/sharp-linux-x64@0.33.2: The engine "pnpm" appears to be invalid.
warning @img/sharp-linux-x64@0.33.2: The engine "glibc" appears to be invalid.
warning @img/sharp-linuxmusl-arm64@0.33.2: The engine "pnpm" appears to be invalid.
warning @img/sharp-linuxmusl-arm64@0.33.2: The engine "musl" appears to be invalid.
warning @img/sharp-linuxmusl-x64@0.33.2: The engine "pnpm" appears to be invalid.
warning @img/sharp-linuxmusl-x64@0.33.2: The engine "musl" appears to be invalid.
warning @img/sharp-wasm32@0.33.2: The engine "pnpm" appears to be invalid.
warning @img/sharp-win32-x64@0.33.2: The engine "pnpm" appears to be invalid.
warning @img/sharp-darwin-x64@0.33.2: The engine "pnpm" appears to be invalid.
warning @img/sharp-darwin-x64@0.33.2: The engine "glibc" appears to be invalid.
warning @img/sharp-libvips-darwin-arm64@1.0.1: The engine "pnpm" appears to be invalid.
warning @img/sharp-libvips-darwin-arm64@1.0.1: The engine "macos" appears to be invalid.
warning @img/sharp-win32-ia32@0.33.2: The engine "pnpm" appears to be invalid.
warning @img/sharp-libvips-linux-x64@1.0.1: The engine "pnpm" appears to be invalid.
warning @img/sharp-libvips-linux-x64@1.0.1: The engine "glibc" appears to be invalid.
warning @img/sharp-libvips-linuxmusl-x64@1.0.1: The engine "pnpm" appears to be invalid.
warning @img/sharp-libvips-linuxmusl-x64@1.0.1: The engine "musl" appears to be invalid.
warning @img/sharp-linux-s390x@0.33.2: The engine "pnpm" appears to be invalid.
warning @img/sharp-linux-s390x@0.33.2: The engine "glibc" appears to be invalid.
[3/4] 🔗  Linking dependencies...
warning "nuxt > @nuxt/devtools@1.0.8" has unmet peer dependency "vite@*".
warning "nuxt > @nuxt/devtools > @nuxt/devtools-kit@1.0.8" has unmet peer dependency "vite@*".
warning "nuxt > @nuxt/devtools > vite-plugin-inspect@0.8.1" has unmet peer dependency "vite@^3.1.0 || ^4.0.0 || ^5.0.0-0".
warning "nuxt > @nuxt/devtools > vite-plugin-vue-inspector@4.0.2" has unmet peer dependency "vite@^3.0.0-0 || ^4.0.0-0 || ^5.0.0-0".
```

**Comment by mateusbrg** [comment]
I had the same problem when following the installation guide as per the documentation. When running the command `yarn add @nuxt/image`, the package is inserted into the package.json `dependencies`. However, when reading the "[Edge Channel](https://image.nuxt.com/get-started/installation#edge-channel)" section, I noticed that the example package is in `devDependencies`. 

So I applied the following:

- Remove lockfile (`package-lock.json`, `yarn.lock`, or `pnpm-lock.yaml`)
- Change the package to devDependencies like this:
```
{
  "devDependencies": {
    "@nuxt/image": "^1.0.0"
  }
}
```
- Reinstall the packages again and run project
